### PR TITLE
Validation error added for bulk user resgistration form

### PR DESCRIPTION
**Pull request by ifahadashraf**

### CODE REVIEW

File: openedx/features/qverse_features/registration/validators.py [hunk 1]
```diff
@@ -17,26 +17,25 @@ def validate_admission_file(file):
...
-        raise ValidationError('', code='invalid')
+        raise ValidationError('Invalid file format. Only csv files are supported.')
 
     FIELD_NAMES = [
                 'regno', 'firstname', 'surname', 'othername', 'levelid',
                 'programmeid', 'departmentid', 'mobile', 'email'
                 ]
-    header_row = []
     try:
+        import pdb; pdb.set_trace();
```
Comment (mumarkhan999) [inline openedx/features/qverse_features/registration/validators.py]: Kindly, remove this breakpoint.

File: openedx/features/qverse_features/registration/validators.py [hunk 2]
```diff
@@ -17,26 +17,25 @@ def validate_admission_file(file):
...
         except Error:
             header_row = get_file_header_row(file_content, 'utf-16')
 
     except Exception:
-        raise ValidationError('', code='invalid')
+        raise ValidationError('Invalid file format. Only utf-8 and utf-16 supported.')
 
     if not all([field_name in header_row for field_name in FIELD_NAMES]):
-        raise ValidationError('', code='invalid')
+        raise ValidationError('Invalid Content. Required columns missing.')
```
Comment (HamzaIbnFarooq) [inline openedx/features/qverse_features/registration/validators.py]:
```suggestion
        raise ValidationError('Invalid content. Required columns are missing.')
```

File: openedx/features/qverse_features/registration/validators.py [hunk 3]
```diff
@@ -17,26 +17,25 @@ def validate_admission_file(file):
...
         file_content = file.read()
-
         try:
             header_row = get_file_header_row(file_content, 'utf-8')
         except Error:
             header_row = get_file_header_row(file_content, 'utf-16')
 
     except Exception:
-        raise ValidationError('', code='invalid')
+        raise ValidationError('Invalid file format. Only utf-8 and utf-16 supported.')
```
Comment (mumarkhan999) [inline openedx/features/qverse_features/registration/validators.py]:
```suggestion
        raise ValidationError('Invalid file encoding format. Only utf-8 and utf-16 file encoding formats are supported.')
```

File: openedx/features/qverse_features/registration/validators.py [hunk 4]
```diff
@@ -45,7 +44,7 @@ def validate_admission_file(file):
         header_row.remove('status')
 
     if len(header_row) != len(FIELD_NAMES):
-        raise ValidationError('', code='invalid')
+        raise ValidationError('Invalid Content. Remove extra columns.')
```
Comment (HamzaIbnFarooq) [inline openedx/features/qverse_features/registration/validators.py]:
```suggestion
        raise ValidationError('Invalid content. Remove extra columns.')
```

**Review by HamzaIbnFarooq — DISMISSED** [review]

File: openedx/features/qverse_features/registration/validators.py (hunk 2 above)
Comment (mumarkhan999) [inline openedx/features/qverse_features/registration/validators.py]:
```suggestion
        raise ValidationError('Invalid Content. Required columns are missing.')
```

**Review by imhassantariq — DISMISSED** [review]

**Review by mumarkhan999 — APPROVED** [review]

**Pull request merged**